pages/home.js

```diff
@@ -1,11 +1,17 @@
 import Layout from '../components/organisms/Layout'
+import SearchCard from '../components/molecules/SearchCard'
 import { getLocalTopics } from './api/getData'
 import TopTasks from '../components/molecules/TopTasks'
 import { MostRequestedList } from '../components/molecules/MostRequestedList'
 
 export default function Home(props) {
   return (
     <Layout locale={props.locale} title="home">
+      <div div className="flex flex-col sm:flex-row w-full">
```

Comment:
Theres a extra "div". I think thats a typo. 

As for the Search button. We can ignore it. I think its my browser. that is acting wierd.

Same for the 50-50 thing. lets leave it as is :)